libretroshare/src/pgp/pgphandler.cc

```diff
@@ -1978,7 +1978,7 @@ bool PGPHandler::removeKeysFromPGPKeyring(const std::set<RsPgpId>& keys_to_remov
 	char template_name[_pubring_path.length()+8] ;
 	sprintf(template_name,"%s.XXXXXX",_pubring_path.c_str()) ;
 	
-	if(mktemp(template_name) == NULL)
+    if(mkstemp(template_name) == -1)
```

Comment:
That's a valid point. I removed this line from my branch. However this function should be fixed because mktemp is insecure (see http://man7.org/linux/man-pages/man3/mktemp.3.html sections BUGS) and it creates nasty warings. Maye we can just replace it with http://man7.org/linux/man-pages/man3/mkdtemp.3.html which does not create a file descriptor.